shared/helpers.js

```diff
@@ -70,3 +71,15 @@ export function translate(format, bindings = {}) {
 
     return ret;
 }
+
+export function currencyExists(currency) {
+    return typeof currencies.find(curr => curr.symbol === currency) !== 'undefined';
+}
+
+export function formatCurrency(currency) {
```

Comment:
How about:
- renaming this function `makeFormatCurrency`
- rename all the `currencyFormatter` in every file `formatCurrency`

It sounds nice, because it will show that what is currently called `currencyFormatter` actually is a function, by having a verb in the name.